app/components/SCForm.js

```diff
@@ -14,7 +8,9 @@ import * as sc from 'react-native-spatialconnect';
 import PlaceHolder from './PlaceHolder';
 import palette from '../style/palette';
 import { buttonStyles } from '../style/style';
-
+import * as ErrorMessages from './rules';
```

Comment:
These are the rules, not the error messages. Change the import name to `Rules` or `FieldRules`